pkg/cluster/cluster.go

```diff
@@ -172,7 +173,7 @@ func (c *Cluster) setStatus(status acidv1.PostgresStatus) {
 }
 
 func (c *Cluster) isNewCluster() bool {
```

Comment:
Maybe we could also introduce similar functions for all the statuses? So that long and tedious ```postgresStatus.PostgresClusterStatus != ClusterStatusSyncFailed``` will become ```!isSyncFailed()```?